### ISSUE TITLE
Multiple Notes about Update

### ISSUE THREAD

**Issue by M4DMAXX** [rauenzi/BDPluginLibrary#186]
Got multiple Times Note about the v2.0.14 Update since 1 Month or more
And it never work at all... it stay on v2.0.13 instead.
I got the Update Note several times a day and it never work.

Anyone know what is happening and how to fix it? 
Thx :)

**Comment by GentlePuppet** [comment]
I've never seen the Zeres plugin library successfully auto-update in like 4 versions, since 2.0.10.

You can manually fix it by deleting the plugin and reinstalling the latest version.

I don't know why this plugin can't auto-update correctly when a dozen other plugins do it just fine.

**Comment by mrwallace888** [comment]
The plugin is permanently stuck on 2.0.14 and I can't get it to 2.0.15, even through manually installing.

This shit is pissing me off.

**Comment by M4DMAXX** [comment]
> I'm tired of having to reinstall this plugin every single time an update comes out. Ever since 2.0.10 it just refuses to update, no matter how many times it asks me of a new version.
> 
> This is getting on my nerves.

Yeah i feel it... on any Update 1. Delete Plugin 2. Download New Version on Github 3. Replace and reload... and Update 2.0.13-15 came out in a short time

**Comment by GentlePuppet** [comment]
Fixing the bug that the auto update failing to update was closed as not planned?

Was zere's built in auto update removed or disabled?

I'll check back if the next update 2.0.17 fails to auto update as well.

Or is it just not a big enough issue that the plugin never actually updates and people will constantly make new ticket's about "too many updates", because they are clueless and unaware that the update actually fails so it keeps asking to update?

**Comment by M4DMAXX** [comment]
Lol I thought the same. Closeing a Update Bug/issue as not planned? Not planned to fix? Or what lol

**Comment by GentlePuppet** [comment]
Got prompted to auto update v2.0.16 to version v2.0.17.
Clicked the update now button.
Failed to update.

This still doesn't work and it was closed as not planned to fix. If yer not going to fix it then remove auto update button that doesn't work.

**Comment by JasonHorkles** [comment]
IMO the built in updater should just be removed entirely if it's broken and the dev won't fix it since BD and PluginRepo both actually work anyways

**Comment by M4DMAXX** [comment]
Yeah Sounds like a solution.

**Comment by Technetium1** [comment]
@JasonHorkles This was already done here: https://github.com/rauenzi/BDPluginLibrary/commit/afbde5e30463a8cf5f2a7849ca4b7fe50d60b80d#diff-6c244fbc1d4eeacaa7a9d351c5a5d525a824e3e846377781ad78405db720323fR98-R3651

**Comment by GentlePuppet** [comment]
After manually updating to v17 I've had it pop up the update notif twice so far to update from v16 to v17. Not sure what that's about.

**Comment by JasonHorkles** [comment]
Same I actually just got the notif

**Comment by rastafaa** [comment]
yep it's still nonstop update request to .17 even though .17 is already the installed version

**Comment by rauenzi** [comment]
If it still happens, update the plugin manually and restart Discord
